.github/workflows/pr_benchmarks.yml

```diff
@@ -46,8 +45,6 @@ jobs:
 
           ./bench.sh run tpch
           ./bench.sh run tpch_mem
-          ./bench.sh run tpch10
```

Comment:
Sorry, my bad, I meant only tpch_mem10 here and below as well (so this should remain). Actually let me open a quick PR myself.

Reply:
This is actually what I had in mind, sorry for the confusion again: https://github.com/apache/arrow-datafusion/pull/9858